bigbluebutton-html5/imports/ui/components/panel-manager/component.jsx

```diff
@@ -152,25 +152,32 @@ class PanelManager extends Component {
 
     if (userListSize.width !== oldUserListSize.width && userListSize.width !== userlistWidth) {
       this.setUserListWidth(userListSize.width);
+      this.forceUpdate();
```

Comment:
AFAIK we should be trying to avoid the use of `this.forceUpdate`.  Is there another alternative to this?

Reply:
In this case I don't think so, I need to do this to force another update after set the width by layout manager.